test/functional/repository/find-options-locking/find-options-locking.ts

```diff
@@ -135,7 +136,7 @@ describe("repository > find options > locking", () => {
                     const originalQuery = entityManager.queryRunner!.query.bind(
                         entityManager.queryRunner,
                     )
-                    entityManager.queryRunner!.query = (...args: any[]) => {
+                    entityManager.queryRunner!.query = (...args: Parameters<QueryRunner['query']>) => {
```

Comment:
This test should have used spies + called with argument. I'm not very familiar with Chai/SinonJS, so I'd rather not change it myself.